### ISSUE TITLE
Support for WebCal protocol

### ISSUE THREAD

**Issue by ScrumpyJack** [texxasrulez/old_calendar#8]
Would be nice to be able to subscribe to webcal calendars, even if it's just to be able to read them

**Comment by skug67** [comment]
Agreed.  And I think the projects which predated this one allowed for that. So it seems like re-enabling it shouldn't be too hard (says the blockhead who has no idea if that's true or not --apologies if I'm just showing my utter lack of understanding the plumbing here and all the work that has gone into it; I certainly appreciate this project with or without webcal support).
